consensus/XDPoS/utils/timeout_pool.go

```diff
@@ -0,0 +1,38 @@
+package utils
+
+import "github.com/XinFinOrg/XDPoSChain/common"
+
+type TimeoutPool struct {
+	hashToTimeout map[common.Hash]*Timeout
+	currentRound  Round
+	certSize      int
+}
+
+func NewTimeoutPool(certSize int) *TimeoutPool {
+	return &TimeoutPool{hashToTimeout: make(map[common.Hash]*Timeout), currentRound: 0, certSize: certSize}
+}
+
+func (pool *TimeoutPool) AddTimeout(timeout *Timeout) *TimeoutCert {
```

Comment:
better just have `t` as indicate it self.
(pool *TimeoutPool) -> (t *TimeoutPool)